contributing.md

```diff
@@ -0,0 +1,18 @@
+# Contributing Guide
+
+## Preparation
+To compile and debug the application it's necessary to install [dotnet 5 sdk](https://dotnet.microsoft.com/en-us/download/dotnet/5.0). It's possible to use any compliant tools for coding and debugging.
```

Comment:
Are there any plans for .NET 6?

Reply:
Not planned yet, but should be simple to implement